raiden/api/rest.py

```diff
@@ -151,6 +152,10 @@ def _add_default_resources(self):
             ConnectionsResource,
             '/connection/<hexaddress:token_address>'
         )
+        self.flask_app.add_url_rule('/', '', self._serve_webui, methods=['GET'])
```

Comment:
You should use something like:
```
self.flask_app.add_url_rule('/<file>', 'index', self._serve_webui, methods=['GET'])
```
For it to pass the `file` parameter.
Please, put it in constructor, below everything else, as `_add_default_resources` should only be used for _resources_, which are API endpoints.